### ISSUE TITLE
Screen Reader is not announcing the expanded state for "User Reference" Collapse/Expand button: A11y_GitHubCopilotExtensionforVSCode_GithubCopilotForAzure_Contrast (Minimum)

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#283391]
Ported from @charris-msft in https://github.com/microsoft/vscode-copilot-issues/issues/316

Source issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2536105
Opening here as this is a VS Code issue, not a GitHub Copilot for Azure issue.

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
Environment Details:
Application Name: Azure GitHub Copilot Extension for VS Code
App used: Visual studio code
VS Code version: 1.102.2(user setup)
Edition Windows 11 Pro
Version 24H2
OS build 26100.3194
Repro Steps:
Open GitHub copilot for Azure.
Github copilot screen will open.
Tab till GitHub copilot chatbot and ask something for ex-@azure Does azure storage data plan support for token based authentication for all data plane APIs in the file service yet? if so, what API version do I need to target?.
Copilot will come with the answer with references.
Observe the issue, Screen Reader is not announcing the expanded state for "User Reference" Collapse/Expand button
Actual Result:
Screen Reader is not announcing the expanded state for "User Reference" Collapse/Expand button.
Issue: Similar issue is observed throughout the application
Expected Result:
Screen Reader should announce the expanded state for "User Reference" Collapse/Expand button.
User Impact:
Screen reader will not be able to understand the state of the button after pressing enter on it.
Recommendations:
Refer [Developer Resources (sharepoint.com)](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx?ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-mchougule%40microsoft.com&OR=Teams-HL&CT=1701062896637&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzExMDIyNDcwMyIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3d%3d)
MAS Reference:
[MAS 4.2.2 – Modifications of States and Properties](https://aka.ms/MAS4.2.2)
